configs/examples.tsconfig.json

```diff
@@ -7,6 +7,6 @@
   },
   "files": ["../examples/app.ts"],
   "exclude": [
-    "**/*.spec.ts", "**/*.spec.tsx"
+    "**/*.spec.ts", "**/*.spec.tsx", "src/util/test-heleper.ts"
```

Comment:
```suggestion
    "**/*.spec.ts", "**/*.spec.tsx", "src/util/test-helper.ts"
```

Reply:
Sorry for the some careless commits. I have committed fix of typos together.